site/themes/docsy/assets/vendor/bootstrap/README.md

```diff
@@ -54,7 +54,6 @@ Read the [Getting started page](https://getbootstrap.com/docs/4.1/getting-starte
 ## Status
 
 [![Slack](https://bootstrap-slack.herokuapp.com/badge.svg)](https://bootstrap-slack.herokuapp.com/)
-[![Build Status](https://img.shields.io/travis/twbs/bootstrap/v4-dev.svg)](https://travis-ci.org/twbs/bootstrap)
```

Comment:
I think this line is irrelevant from kustomize. It's from dependency.

Reply:
good catch.  Let me revert this change